### PR TITLE
Common names in gray

### PR DESCRIPTION
**Pull request by jameshowardwang**
# Description

Make non-valid common names gray when viewing as common name

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix a feature that knowingly causes existing functionality to not work as expected)

# Test and Reproduce
Outline the steps to test or reproduce the PR here.

1. Navigate to phylo tree
2. Toggle between scientific and common name
3. Verify that common name are black and scientific names are gray
4. Verify other filters, etc. still work

## Impacted Areas in Application
List general components of the application that this PR will affect:

Which pages?

- [ ] Login Screen
- [ ] Single Upload Page
- [ ] Batch Upload Page
- [ ] All Projects Page
- [ ] Sample Details Page
- [x] Report Page

Specific components 

* 

# Testing Script:

###Login Page
* Successfully login to the application and get taken to the project page

###Single Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Uploading a duplicate file throws an error
* Pipeline gets initiated successfully 

###Batch Upload
* Using an AWS path that doesn't exist throws an error
* Succesfully uploaded files from a folder (use test host)
* Navigate to batch upload settings page after upload 
* Host selection from previous page still selected 
* Uploading a duplicate file throws a warning
* Pipeline gets initiated successfully 

###All Project Page
* Switch between projects 
* Search for a sample
* Navigate through different pages of samples
* Select Upload new sample button
* Select sample to see generated report

###Sample Details Page
* Information is consistent from Project Page
* Notes are editable and save successfully 
* Can navigate to report page 

###Report Page
* Data loads succesfully 
* Collapse and Uncollapse button works
* Clicking on the taxonomy name gives you total reads for that taxon
* Category filter works
* Genus Search works
* Filters work as expected
* Disabling Filters works
* Filtering by column works 

# Checklist:

- [x] I have run through the testing script to make sure current functionality is unchanged
- [x] I have done relevant tests that prove my fix is effective or that my feature works
- [x] I have spent time testing out edge cases for my feature
- [x] I have updated the test script or pull request template if necessary
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/assets/src/components/PipelineSampleTree.jsx [hunk 1]
```diff
@@ -78,13 +81,15 @@ class PipelineSampleTree extends React.PureComponent {
...
           name;
 
-        if (this.props.nameType == "Common name") {
-          name = row.lineage[level + "_common_name"];
-        }
-
-        if (!name) {
+        if (
+          this.props.nameType == NAME_TYPE_COMMON_NAME &&
+          (name = row.lineage[level + "_common_name"])
```
Comment (jshoe) [inline app/assets/src/components/PipelineSampleTree.jsx]: wait did you mean 'name == row.lineage'? or should that line be in the 'if' block?

**Comment by jameshowardwang** [comment]
I meant it this way but it’s arguably confusing. I’m doing a set and a
check if empty in same line.
